src/profile-logic/data-structures.js

```diff
@@ -295,8 +295,10 @@ export function getEmptyExtensions(): ExtensionTable {
 
 export function getDefaultCategories(): CategoryList {
   return [
-    { name: 'Idle', color: 'transparent', subcategories: ['Other'] },
+    // Make sure 'Other' is at index 0, as it's used as the category for stacks when no
+    // categories are provided by an imported (non-Gecko profiler) profile.
     { name: 'Other', color: 'grey', subcategories: ['Other'] },
+    { name: 'Idle', color: 'transparent', subcategories: ['Other'] },
```

Comment:
Yikes. This little change invalidated lots of test snapshots and assertions. I believe most of them were transitory.